contracts/colonyNetwork/ColonyNetworkSkills.sol

```diff
@@ -210,27 +219,35 @@ contract ColonyNetworkSkills is ColonyNetworkStorage, Multicall {
 
     (bool success, ) = miningBridgeAddress.call(payload);
     require(success, "colony-network-bridging-tx-unsuccessful");
+
+    emit ReputationUpdateSentToBridge(_colony, _updateNumber);
   }
 
   // Bridging (receiving)
 
+  modifier skillAndBridgeConsistent(address bridgeAddress, uint256 _skillId) {
```

Comment:
I might've called this `validBridgeTx`

Reply:
I thought along those lines too originally, but in one case, it's not a function that's called from a bridge, it's a function that's called when something's been bridged out of order.